remove-unused-feature-flags/test.ts

```diff
@@ -50,16 +54,24 @@ describe('remove-unused-feature-flags', function () {
 
 	it('should remove a feature flag check within Promise.all() (with options)', function () {
 		const INPUT = `
-        const [a, b] = await Promise.all([
+        const [b, a] = await Promise.all([
+			fnc('b'),
```

Comment:
any reason that this comes before `Promise.resolve('a'),`?

Reply:
yes, this is testing the removal of the variable names in different positions in the array